src/unix/bsd/freebsdlike/dragonfly/mod.rs

```diff
@@ -1071,6 +1071,10 @@ extern "C" {
         clk_id: ::clockid_t,
         tp: *const ::timespec,
     ) -> ::c_int;
+    pub fn clock_getcpuclockid(
+        pid: ::pid_t,
+        clk_id: *mut ::clockid_t,
+    ) -> ::c_int;
```

Comment:
I think you can just place it on `freebsdlike/mod.rs`?

Reply:
Sure, let me try that.

Reply:
Should all other `clock_*` functions be in `freebsdlike/mod.rs` too then? Looks like the only difference between freebsd and dragonfly is that one uses `c_int` for `clockid_t` and the other `c_ulong`. Does it make sense to have functions in  `freebsdlike/mod.rs`, while `clockid_t` in more platform specific place?

Reply:
> Should all other clock_* functions be in freebsdlike/mod.rs too then?

Yeah, sounds good.

> Does it make sense to have functions in freebsdlike/mod.rs, while clockid_t in more platform specific place?

It should be fine as we `pub use` all the items.

Reply:
Done!